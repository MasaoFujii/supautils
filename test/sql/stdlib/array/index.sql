select supa.index(array['a','b','c'], 'c');

select supa.index(array[1, 2, 3], 4);

select supa.index(array[]::int[], 1);

select supa.index(null::int[], 1);
