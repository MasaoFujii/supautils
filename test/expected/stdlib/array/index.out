select supa.index(array['a','b','c'], 'c');
 index 
-------
     3
(1 row)

select supa.index(array[1, 2, 3], 4);
 index 
-------
      
(1 row)

select supa.index(array[]::int[], 1);
 index 
-------
      
(1 row)

select supa.index(null::int[], 1);
 index 
-------
      
(1 row)

